src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -33,6 +34,8 @@
     public static final String VALID_IC_BOB = "T0102030G";
     public static final String VALID_NAME_AMY = "Amy Bee";
     public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_DOB_AMY = "14-02-1997";
+    public static final String VALID_DOB_BOB = "11-11-2001";
```

Comment:
Perhaps it is better to name it VALID_DATE_OF_BIRTH_AMY and VALID_DATE_OF_BIRTH_BOB to keep it consistent with the current style, where the full class name is used instead of abbreviations?